ArduPlane/mode.h

```diff
@@ -296,17 +296,32 @@ class ModeGuided : public Mode
     // handle a guided target request from GCS
     bool handle_guided_request(Location target_loc) override;
 
+    // handle a guided path following request
+    bool handle_guided_path_request(const Location& location_on_path, const Vector2f& unit_path_tangent, const float path_curvature, const bool direction_is_ccw);
```

Comment:
Could we change this to allow positive or negative curvature, and then remove `direction_is_ccw`? less params is nice.

Reply:
Less params is nice, but using the flag for direction is more consistent with a number of other methods that break it out like this.